upgrading/index.adoc

```diff
@@ -81,6 +81,18 @@ in-place upgrades or blue-green deployments.
 The manual upgrade method breaks down the steps that happen during an automated
 Ansible-based upgrade and provides the equivalent commands to run manually. Using this method describes the in-place xref:install-config-upgrading-strategies[upgrade strategy].
 
+[[upgrading-swap-memory]]
+== Considerations for upgrading with swap memory
+
+If you are upgrading an existing, older version of an {product-title} cluster,
```

Comment:
I might say:
If you are upgrading an existing, older version of an {product-title} cluster,
you might have swap memory enabled. You can enable swap memory by setting the `openshift_disable_swap` parameter in the Ansible inventory file to `false` or manually enabling swap
in the cluster. If you enabled swap memory, then upgrading to {product-title} version 3.9 will
fail. To successfully upgrade to {product-title} version 3.9, ensure swap memory
is disabled by following the
xref:../../admin_guide/overcommit.adoc#disabling-swap-memory[Disabling Swap
Memory] section of the Cluster Administration guide.